src/hardware/lpt_dac.cpp

```diff
@@ -38,7 +40,13 @@ LptDac::LptDac(const std::string_view name, const int channel_rate_hz,
 	using namespace std::placeholders;
 
 	assert(!dac_name.empty());
-	const auto audio_callback = std::bind(&LptDac::AudioCallback, this, _1);
+
+	constexpr bool Stereo = true;
+	constexpr bool SignedData = true;
+	constexpr bool NativeOrder = true;
+	const auto audio_callback = std::bind(MIXER_PullFromQueueCallback<LptDac, AudioFrame, Stereo, SignedData, NativeOrder>,
+	                                std::placeholders::_1,
+	                                this);
```

Comment:
Minor: you could run these through clang-format.

Reply:
Fixed